src/main/scala/castalia/model/Model.scala

```diff
@@ -39,17 +40,44 @@ object Model extends DefaultJsonProtocol  {
                                     httpStatusCode: Option[StatusCode],
                                     response: Option[AnyJsonObject])
 
-  /**
+   /**
     * Specification of the response latency to simulate
-    * @param distribution String with type of distribution
+     *
+     * @param distribution String with type of distribution
     * @param mean String with mean (quantity + time unit)
     */
-  case class LatencyConfig(distribution:String, mean:String) {
-    def duration: Duration = Duration(mean)
-  }
+    case class LatencyConfig(distribution: String, mean: Option[String], p50: Option[String], p90: Option[String])
+     extends DelayedDistribution {
+
+      def duration: Duration = Duration(mean.getOrElse("0 ms"))
+
+      val dist: Distribution[Double] = {
+
+        val p1 = Duration(p50.getOrElse("0 ms")).toMillis
+        val p2 = Duration(p90.getOrElse("0 ms")).toMillis
+
+        distribution match {
+          case "gamma" =>
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/scala-academy/Castalia/file/2016627755/issues/source?bid=2998213&fileBranchId=3112713#l61)